src/features/user/controllers/user-controller.test.ts

```diff
@@ -132,4 +145,69 @@ describe('[Controllers] UserController', () => {
       expect(next).toHaveBeenCalledWith(expect.any(Error));
     });
   });
+
+  describe('getAccounts', () => {
+    it('should call service with correctly params', async () => {
+      const { req, res, next, userController, getUserAccountsService } =
+        makeSut();
+      const serviceSpy = vi.spyOn(getUserAccountsService, 'execute');
+      const uuid = randomUUID();
+      req.params = {
+        id: uuid,
+      };
+
+      await userController.getAccounts(req, res, next);
+
+      expect(serviceSpy).toHaveBeenCalledWith({ id: uuid });
+    });
```

Comment:
Acho que não precisa desse teste já que o `'should return accounts from user` que é um teste de sucesso  contempla esse exemplo